psi4/driver/molutil.py

```diff
@@ -252,6 +256,11 @@ def geometry(geom, name="default"):
         geom, enable_qm=True, missing_enabled_return_qm='minimal', enable_efp=True, missing_enabled_return_efp='none')
 
     molecule = core.Molecule.from_dict(molrec['qm'])
+    if "geom" in molrec["qm"]:
+        geom = np.array(molrec["qm"]["geom"]).reshape((-1, 3))
+        if molrec["qm"]["units"] == "Angstrom":
```

Comment:
`.title() == "Angstrom"` to be safe or lower it?

Reply:
thanks for the catch, but this is psi4 schema, and it's strictly `Angstrom|Bohr`

Reply:
I always get a little nervous around these since the data is “just” a dict and there is no code-ensured contract for title case.